packages/blockchain-wallet-v4-frontend/src/modals/Swap/PreviewSwap/PreviewSwapSuccess.tsx

```diff
@@ -0,0 +1,286 @@
+                  size='16px'
+                  color={isActiveExchangeToolTip ? 'blue600' : 'grey300'}
+                  onClick={toggleExchangeTooltip}
+                />
+              </IconWrapper>
+            </ExchangeRateRow>
+          </Title>
+          <Value data-e2e='swapExchangeRate' style={{ marginTop: 0 }}>
+            <Text weight={400} color='grey900'>
+              1 {baseCoinDisplaySymbol} = {formatCoin(props.quote.price)} {counterCoinDisplaySymbol}
+            </Text>
+          </Value>
+        </div>
+        {isActiveExchangeToolTip && (
+          <ToolTipText>
+            <Text size='12px'>
+              <TextGroup inline>
+                <FormattedMessage
+                  id='copy.swap_exchange_rate_tt_1'
+                  defaultMessage='The exchange rate is the best price available for'
+                />{' '}
+                {counterCoinDisplaySymbol}{' '}
+                <FormattedMessage
+                  id='copy.swap_exchange_rate_tt_2'
+                  defaultMessage='in terms of 1'
+                />{' '}
+                {baseCoinDisplaySymbol}{' '}
+                <Link
+                  href='https://support.blockchain.com/hc/en-us/articles/360061672651'
+                  size='12px'
+                  rel='noopener noreferrer'
+                  target='_blank'
+                >
+                  <FormattedMessage
+                    id='modals.simplebuy.summary.learn_more'
+                    defaultMessage='Learn more'
+                  />
+                </Link>
+              </TextGroup>
+            </Text>
+          </ToolTipText>
+        )}
+      </StyledRow>
+      <Row style={{ borderTop: '0' }}>
+        <FeeBreakdownBox
+          counter={COUNTER}
+          counterQuote={Remote.Success(props.quote)}
+          base={BASE}
+          basePayment={Remote.Success(props.payment)}
+        />
+        <TextGroup inline style={{ marginTop: '16px', textAlign: 'center' }}>
+          <Text size='12px' weight={500} color='grey600'>
+            <FormattedMessage
+              id='copy.swap_amount_change_disclaimer'
+              defaultMessage='Final amount may change due to market activity. By approving this Swap you agree to Blockchain.com’s'
+            />
+          </Text>
+          <Text size='12px' weight={500} color='grey600'>
+            <Link
+              href='https://support.blockchain.com/hc/en-us/articles/4417063009172'
+              size='12px'
+              rel='noopener noreferrer'
+              target='_blank'
+            >
+              <FormattedMessage id='copy.refund_policy' defaultMessage='Refund Policy' />.
+            </Link>
+          </Text>
+        </TextGroup>
+      </Row>
+      <FlyoutWrapper>
+        <Form onSubmit={handleSubmit}>
+          <Button
+            nature='primary'
+            data-e2e='swapBtn'
+            type='submit'
+            disabled={props.submitting}
```

Comment:
We need to disable this button when quote is refreshing. You can refer to `SellButton.tsx` for an example.

Reply:
`props.submitting || isCompletingSoon`